sky/packages/sky/lib/src/widgets/basic.dart

```diff
@@ -1095,6 +1097,39 @@ class DefaultAssetBundle extends InheritedWidget {
   bool updateShouldNotify(DefaultAssetBundle old) => bundle != old.bundle;
 }
 
+class RawImage extends StatelessComponent {
+  RawImage(
+      {Key key,
+      this.bytes,
+      this.width,
+      this.height,
+      this.colorFilter,
+      this.fit,
+      this.repeat: ImageRepeat.noRepeat,
+      this.centerSlice})
```

Comment:
The style guide says to format the whitespace here a bit differently.  See `AssetImage` for the style we're using.